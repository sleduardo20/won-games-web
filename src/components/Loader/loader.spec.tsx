import { renderWithTheme } from 'utils/tests/helpers';

import { Loader } from '.';

describe('<Loader />', () => {
  it('should be able render Loader correctly', () => {
    const { container } = renderWithTheme(<Loader />);

    expect(container).toMatchSnapshot();
  });
});
