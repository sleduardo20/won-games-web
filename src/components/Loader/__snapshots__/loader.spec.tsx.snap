// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should be able render Loader correctly 1`] = `
.c0 {
  border: 16px solid #fafafa;
  border-radius: 50%;
  border-top: 16px solid #f231a5;
  width: 120px;
  height: 120px;
  -webkit-animation: spin 1.3s linear infinite;
  -webkit-animation: spin 1.3s linear infinite;
  animation: spin 1.3s linear infinite;
}

<div>
  <div
    class="c0"
  />
</div>
`;
